### DISCUSSION TITLE
Ability to disable "tasks"

### DISCUSSION THREAD

**Discussion by codetalker78** [kromitgmbh/titra#73] [Ideas]
Hi,
our use case for time tracking is to track hours per project as simply as possible. 
For us, defining a task for each time entry is not needed. But this seems to be mandatory, which increases the effort to add a time record unnecessarily.

Is it possible to disable the requirement to add a task?

A related issue: 
We like the week view very much for efficient time entry. Having one standard-task per project and being able to tab-switch between the fields is near perfect, but switching to the next week, the prior task lines are not available any more and must be recreated. It would be nice, if they would persist over week changes.

**Comment by faburem** [comment]
I really like the idea! titra has been built all around simplicity. However removing the task completely would be a pretty big change, what do you think about setting a default task for projects that is automatically pre-filled when you track time?

**Comment by codetalker78** [comment]
Yes, that would be a good solution. Defaults allow to ignore it, when not needed and to use it when relevant.

**Comment by faburem** [comment]
The default task feature has shipped in titra version 0.58.0. Please give it a try and let me know what you think!

Thanks,
Fabian
